spec/mongoid/config_spec.rb

```diff
@@ -197,6 +197,34 @@
     end
   end
 
+  context 'when discriminator_key option' do
```

Comment:
This sentence fragment is missing a verb, alternatively you can say something like  `context 'discriminator_key override'`.